ReactTask/src/components/CreateAppointment/CreateAppointment.js

```diff
@@ -21,9 +21,13 @@ const CreateAppointment = ({ switchContent }) => {
   const [isChooseAllData, setChooseAllData] = useState(false),
     [isFilledData, setFilledData] = useState(false),
     [isPossibleTimeArr, setPossibleTimeArr] = useState([]),
+    [isNoteUser, setNoteUser] = useState('stomach-ache'),
+    [isTimeChooseCheckbox, setTimeChooseCheckbox] = useState(''),
     [isTimeSelect, setTimeSelect] = useState(''),
-    [isChooseUser, setChooseUser] = useState(''),
-    [isDoctorChoose, setDoctorChoose] = useState('');
+    [isChooseUser, setChooseUser] = useState('2021-12-22T22:00:00.000Z'),
+    [isDoctorChoose, setDoctorChoose] = useState(
+      '902240b7-514a-48c3-a67f-6acfb1d35030'
+    );
```

Comment:
I am sure it is bad practice to hardcode values. line 27,29

Reply:
I got an error when calling the doctor's time
Deleted